staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -293,6 +293,8 @@ type CPU struct {
 	// Sockets specifies the number of sockets inside the vmi.
 	// Must be a value greater or equal 1.
 	Sockets uint32 `json:"sockets,omitempty"`
+	// MaxSockets
```

Comment:
What's the default value here?

Does `Must be a value greater or equal 1.` apply too?